### ISSUE TITLE
ATI memory loss?

### ISSUE THREAD

**Issue by Ligti** [ValveSoftware/steam-for-linux#3776]
Hello.
When i start the Steam Big Picture Mode and go in to system it shows the GPU men as: -0.0 memory and in the Display option it says ATI tech 256 MB and i had 1088 as default resulution. I run 1920x1080 now with 14.501.1003-141120a-178002C driver. I am running a Intel G3220 8 Gig 2133mhz and AMD ATI HAWAII PRO (Gigabyte R9 290).

**Comment by Ligti** [comment]
Hello i just took a screenshoot of steam and AMD CCC if it will help you. 
I tried a videostress test in CS:go and before the problem started i use to have 568 fps and now 256. It started after the last update. 
![20150415_064400](https://cloud.githubusercontent.com/assets/11732903/7152427/1ce58e14-e33c-11e4-8c77-c08aac2d5f35.jpg)

![20150415_064200](https://cloud.githubusercontent.com/assets/11732903/7152418/edffda64-e33b-11e4-8756-1c0f6add5541.jpg)

**Comment by Ligti** [comment]
I got the same output on my Intel I5 4690k 16 gig ram and HD 7850 



![20150415_103831](https://cloud.githubusercontent.com/assets/11732903/7155071/104014ca-e35c-11e4-8b7b-b057608e43e5.jpg)

**Comment by mtmkls** [comment]
Yeah, it's a known issue that Steam (and almost all games) can't detect vram size on Linux, because there's no standard way to do it.

**Comment by Ligti** [comment]
Oki but that do not explaine the fps drop from 568-256

**Comment by Ligti** [comment]
![568](https://cloud.githubusercontent.com/assets/11732903/7196760/2adf0746-e4d8-11e4-9140-19cfc58dc486.jpg)

**Comment by Ligti** [comment]
and now i get 256 or less with the same computer and parts

**Comment by TierraDelFuego** [comment]
@mtmkls if it is a known issue what is the issue #? 

@gdrewb-valve and @triage-valve a proof of concept to get memory info can be found here:
https://github.com/TierraDelFuego/clinfo-parse

I tried this on brewmaster and it worked fine.

**Comment by TierraDelFuego** [comment]
@Ligti best to file a new issue about the stress-test FPS.
